PathPlanning/DStarLite/d_star_lite.py

```diff
@@ -52,6 +53,8 @@ class DStarLite:
         Node(-1, -1, math.sqrt(2))
     ]
 
+    initialized = False
```

Comment:
This should be initizlied in the `__init__`?

Reply:
Yes, I moved it, thank you